frontend-web/webclient/app/Applications/Studio/Group.tsx

```diff
@@ -215,6 +191,40 @@ export const AppGroup: React.FunctionComponent = () => {
                                               defaultValue={group.data?.specification.description}/>
                                 </Label>
 
+                                <Flex justifyContent="right" mb="30px">
+                                    <Button type="button" onClick={async () => {
+                                        if (!group.data) return;
+
+                                        const titleField = groupTitleField.current;
+                                        if (titleField === null) return;
+
+                                        const newTitle = titleField.value;
+                                        if (newTitle === "") {
+                                            snackbarStore.addFailure("Title cannot be empty", false);
+                                            return
+                                        }
+
+                                        const descriptionField = groupDescriptionField.current;
+                                        if (descriptionField === null) return;
+
+                                        const newDescription = descriptionField.value;
+
+                                        const success = await invokeCommand(AppStore.updateGroup({
+                                            id: group.data?.metadata.id,
+                                            newTitle: newTitle,
+                                            newDescription: newDescription,
+                                            newDefaultFlavor: defaultApplication,
+                                            newLogoHasText: logoHasText
+                                            // tags
```

Comment:
Something that needs to be done?

Reply:
If you mean the `// tags` comment, it was already there. Probably from the old app store, but not sure.

Reply:
Okay, then I'll mark this as resolved.